File: Freedom/微能量/经典案例/EnergyBusinessDetailViewController.m
//  EnergyBusinessDetailViewController.m
//  Created by Super on 16/9/5.
//  Copyright © 2016年 Super. All rights reserved.
//
#import "EnergyBusinessDetailViewController.h"
@implementation EnergyBusinessDetailViewController{
    UIWebView *web;
}
-(void)viewDidLoad{
    [super viewDidLoad];
    NSURL *url = [NSURL URLWithString:@"http://www.taobao.com"];
    NSURLRequest *request = [NSURLRequest requestWithURL:url];
    web = [[UIWebView alloc]initWithFrame:self.view.bounds];
    [web loadRequest:request];
    [self.view addSubview:web];
}
@end

File: Freedom/微能量/经典案例/EnergyBusinessDetailViewController.swift
//
//  EnergyBusinessDetailViewController.swift
//  Freedom
import UIKit
class EnergyBusinessDetailViewController: EnergyBaseViewController {
    override func viewDidLoad() {
        super.viewDidLoad()
        let web = UIWebView(frame:view.bounds)
        let url = URL(string: "http://www.taobao.com")
        let request = URLRequest(url: url!)
        web.loadRequest(request)
        view.addSubview(web)
    }
}
